src/commands/sharinpix/form/json2csv.ts

```diff
@@ -0,0 +1,156 @@
+      command: '<%= config.bin %> <%= command.id %>',
+    },
+  ];
+
+  public async run(): Promise<Json2CsvResult> {
+    const directory = 'sharinpix/forms';
+
+    let entries: string[];
+    try {
+      entries = await fs.promises.readdir(directory);
+    } catch (error) {
+      this.warn(
+        `Failed to read directory ${directory}: ${error instanceof Error ? error.message : 'Unknown error'
+        }`
+      );
+
+      return {
+        name: 'OK',
+      };
+    }
+
+    const jsonFiles = entries.filter((file) => file.endsWith('.json'));
+
+    let processed = 0;
+    let failed = 0;
+    let skipped = 0;
+
+    for (const file of jsonFiles) {
+      const fullPath = path.join(directory, file);
+
+      try {
+        // eslint-disable-next-line no-await-in-loop
+        const content = await fs.promises.readFile(fullPath, 'utf8');
+        const parsed = JSON.parse(content) as { elements?: unknown };
+
+        if (!Array.isArray(parsed.elements)) {
+          this.warn(`File ${file} skipped: no 'elements' array found`);
+          skipped++;
+          continue;
+        }
+
+        const elements = parsed.elements as Array<Record<string, unknown>>;
+
+        if (elements.length === 0) {
+          this.warn(`File ${file} skipped: empty 'elements' array`);
+          skipped++;
+          continue;
+        }
+
+        const headerSet = new Set<string>();
+        for (const element of elements) {
+          for (const key of Object.keys(element)) {
+            headerSet.add(key);
+          }
+        }
+
+        const headers = Array.from(headerSet);
+        const rows: string[] = [];
+
+        // Header row
+        rows.push(this.toCsvRow(headers));
+
+        // Data rows
+        for (const element of elements) {
+          const values: string[] = [];
+
+          for (const key of headers) {
+            const raw = element[key];
+            const cell = this.formatCell(key, raw);
+            values.push(this.escapeCsvValue(cell));
+          }
+
+          rows.push(values.join(','));
+        }
+
+        const csvContent = rows.join('\n');
+        const csvPath = fullPath.replace(/\.json$/i, '.csv');
+
+        // eslint-disable-next-line no-await-in-loop
+        await fs.promises.writeFile(csvPath, csvContent, 'utf8');
+        processed++;
+      } catch (error) {
+        this.warn(
+          `Failed to process file ${file}: ${error instanceof Error ? error.message : 'Unknown error'}`
+        );
+        failed++;
+      }
+    }
+
+    this.log(`CSV generation completed. Processed: ${processed}, failed: ${failed}, skipped: ${skipped}`);
+
+    return {
+      name: 'OK',
+    };
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  private formatCell(key: string, raw: unknown): string {
+    if (raw === null || raw === undefined) {
+      return '';
+    }
+
+    if (key === 'options' && Array.isArray(raw)) {
```

Comment:
Instead of key == 'options' I think in a first step we should just do a json.serialize when the object is an array or a object. Then in a second step we will mark some keys (like options to be serialized in multiple columns options_1_label, options_2_label, .... So that we can preserve the way to edit things in an excel...